src/consistency-testing/gobekli/gobekli/chaos/analysis.py

```diff
@@ -523,29 +534,28 @@ def make_overview_chart(title, log_dir, availability_log, latency_log,
 
     with open(path.join(log_dir, f"overview.lat.{latency_type_name}.log"),
               "w") as chart_lat_file:
-        with open(path.join(log_dir, latency_log)) as latency_log_file:
-            mn = sys.maxsize
-            for line in latency_log_file:
-                if latency_type == LatencyType.PRODUCER:
-                    if "ok" not in line:
-                        continue
-                parts = line.rstrip().split("\t")
-                tick = int(parts[0])
-                if tick < latency_cut_off_us:
+        mn = sys.maxsize
+        for line in read_latency(log_dir, latency_log):
+            if latency_type == LatencyType.PRODUCER:
+                if "ok" not in line:
                     continue
-                tick = int(tick / 1000000)
-                lat = int(parts[latency_type.value])
-                if "ok" in line:
-                    maxtick = max(maxtick, tick)
-                    mn = min(mn, lat)
-                    maxlat = max(maxlat, lat)
-                out = f"{tick}\t{lat}"
-                for i in range(2, len(parts)):
-                    out += "\t" + parts[i]
-                chart_lat_file.write(out + "\n")
-            maxminlat = mn * 3
-            minlatstep = mn
-            maxmaxx = int(1.2 * maxlat)
+            parts = line.rstrip().split("\t")
+            tick = int(parts[0])
+            if tick < latency_cut_off_us:
+                continue
+            tick = int(tick / 1000000)
+            lat = int(parts[latency_type.value])
+            if "ok" in line:
+                maxtick = max(maxtick, tick)
+                mn = min(mn, lat)
+                maxlat = max(maxlat, lat)
+            out = f"{tick}\t{lat}"
+            for i in range(2, len(parts)):
+                out += "\t" + parts[i]
+            chart_lat_file.write(out + "\n")
+        maxminlat = mn * 3
```

Comment:
are there any conditions (it looks like if there aren't any `ok` lines) that control would fall through this loop and `mn`'s initial value of maxint would be  `* 3` and i'm not really sure what that would mean. would it overflow or have some other unintended consequence, like on the next go around the loop?

Reply:
that's true but I think it's fine, the situation is unlikely and the output is only used by people so if there is an overflow they will notice :)